plugins/modules/proxmox_kvm.py

```diff
@@ -482,6 +482,22 @@
       - Timeout for operations.
     type: int
     default: 30
+  tpmstate0:
+    description:
+      - A hash/dictionary of options for the Trusted Platform Module disk.
+      - A TPM state disk is required for Windows 11 installations.
+    suboptions:
+      storage:
+        description:
+          - C(storage) is the storage identifier where to create the disk.
+        type: str
+      version:
+        description:
+          - The TPM version to use.
+        type: str
+        choices: ['1.2', '2.0']
+    type: dict
+    version_added: 7.2.0
```

Comment:
The next minor release will be 7.1.0.
```suggestion
    version_added: 7.1.0
```